hack/do_sync.sh

```diff
@@ -43,7 +43,7 @@ for i in attacher,master provisioner,master resizer,master; do
     (cd pkg/${SIDECAR} && git checkout ${SIDECAR_HASH})
 
     cat pkg/${SIDECAR}/go.mod | grep "	" | grep -v "indirect" >> tmp/gomod-require.txt
-    cat pkg/${SIDECAR}/go.mod | { grep "replace " || [[ $? == 1 ]] } >> tmp/gomod-replace.txt
+    cat pkg/${SIDECAR}/go.mod | { grep "replace " || [[ $? == 1 ]]; } >> tmp/gomod-replace.txt
```

Comment:
CI can build without the semicolon, in which environment are you trying to build the project?

Reply:
I'm failing to build an image in our cloud VM environment, I assume this fix will be fine in CI?